src/VocabGeneration.test.js

```diff
@@ -189,6 +189,31 @@ describe("Suite for generating common vocabularies (marked as [skip] to prevent
   // it("tests a single custom vocab", async () => {
   it.skip("tests a single custom vocab", async () => {
     await generateVocabArtifact({
+      // inputResources: ["http://www.w3.org/ns/locn"],
+
+      // Typo in the Turtle (trailing full stop inside a BNode on line 22!
+      // inputResources: ["http://purl.org/vocab/cpsv#"],
+
+      // inputResources: ["http://www.w3.org/ns/adms#"],
+      // inputResources: ["http://www.w3.org/ns/org#"],
+      // inputResources: ["http://www.w3.org/ns/person#"],
+      // inputResources: ["http://www.w3.org/ns/regorg#"],
+
+      // inputResources: ["https://purl.org/oslo/ns/localgov#"],
+      // vocabContentTypeHeaderOverride: "application/rdf+xml",
+      // ignoreNonVocabTerms: true,
+
+      // inputResources: ["http://www.w3.org/ns/dpv#"],
+      // namespaceOverride: "https://w3id.org/dpv#",
+
+      // inputResources: [
+      //   // "http://purl.org/oslo/ns/localgov/",
+      //   "https://raw.githubusercontent.com/rafbuyle/oslo_xml_schemas/master/oslo-v1_1_0.rdf",
+      // ],
+      // nameAndPrefixOverride: "oslo",
+      // vocabContentTypeHeaderOverride: "application/rdf+xml",
+      // ignoreNonVocabTerms: true,
+      //
```

Comment:
This being a test skipped by default and included in the codebase, and not a script in a separate folder, is really not great. It adds nois to a lot of PRs, and clutters the codebase. The same goes for the end-to-end tests with hard-coded paths in them.

Reply:
Yeah, I know - this whole test file is really just a convenience thing I use during development (i.e., **_all_** the tests in this entire file are `skipped` by default!). I basically use these tests to test out the YAML files in the separate `solid-common-vocab-rdf` repository. But yeah, I should look at moving things around - i.e., just moving this entire test file to that repo prehaps...
The ever-growing list of vocabs in this particular test has been kinda handy though lots of times though, as if I want to rerun the AG for just a single vocab, I can pop in here and just uncomment it and rerun the test (this is handy 'cos almost all vocabs our there in the wild need something overridden (e.g., `vocabContentTypeHeaderOverride:`, or `ignoreNonVocabTerms:`, or `namespaceOverride:`), so having them recorded here has been really handy sometimes).
But that's for another PR I reckon...